src/lib/rnp.cpp

```diff
@@ -6899,6 +6899,21 @@ try {
 }
 FFI_GUARD
 
+rnp_result_t
+rnp_key_is_expired(rnp_key_handle_t handle, bool *result)
+try {
+    if (!handle || !result) {
+        return RNP_ERROR_NULL_POINTER;
+    }
+    pgp_key_t *key = get_key_prefer_public(handle);
+    if (!key) {
+        return RNP_ERROR_BAD_PARAMETERS;
```

Comment:
Codecov says this line is not covered by tests.

Reply:
Hm, wondering how to make it covered... Will think about.